kythe/typescript/indexer.ts

```diff
@@ -30,6 +30,32 @@ const LANGUAGE = 'typescript';
  * code duplication.
  */
 export interface IndexerHost {
+  /**
+   * getSymbolAtLocation is the same as ts.TypeChecker.getSymbolAtLocation,
+   * except that it has a return type that properly captures that
+   * getSymbolAtLocation can return undefined.  (The TypeScript API itself is
+   * not yet null-safe, so it hasn't been annotated with full types.)
+   */
+  getSymbolAtLocation(node: ts.Node): ts.Symbol|undefined;
+  /**
+   * Computes the VName (and signature) of a ts.Symbol. A Context can be
+   * optionally specified to help disambiguate nodes with multiple declarations.
+   * See the documentation of Context for more information.
+   */
+  getSymbolName(sym: ts.Symbol, ns: TSNamespace, context?: Context): VName;
+  /**
+   * Aliases a local symbol to the VNames of a remote symbol.
```

Comment:
What does "aliasing" as a verb here mean?  Records that they are the same in memory, emits something, or something else?

Reply:
Updated the comment.